.storybook/components/CodeViewer/CodeViewer.helpers.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+
+const tab = "  ";
+const newline = "\n";
+
+function tabs(num) {
+  return Array(num)
+    .fill(tab)
+    .join("");
+}
+
+function isArray(content) {
+  return Boolean(content && typeof content !== "string" && content.length !== undefined);
```

Comment:
`Array.isArray(object)` already implemented this or it's this something totally different?